src/screens/Filter/Filter.tsx

```diff
@@ -9,23 +9,17 @@ import { useFilterController } from './useFilterController';
 import { styles } from './styles';
 
 export function Filter(): JSX.Element {
-  const { onSelectPress, onFilterChange, filters } = useFilterController();
+  const { onSelectPress, filters } = useFilterController();
 
   const { bottom } = useSafeAreaInsets();
   const footerOffset = bottom || 20;
 
   return (
     <View style={styles.modalContainer}>
       <ScrollView>
-        {filters.map((filter, index) => {
+        {filters.map(() => {
           return (
-            <FilterItem
-              filter={filter}
-              index={index}
-              key={filter.id}
-              id={filter.id}
-              onChange={onFilterChange}
-            />
+            <FilterItem />
```

Comment:
key property is missed

Reply:
I will realize filter functionality in the future, in another brunch